inspector_go/rpc/req.go

```diff
@@ -0,0 +1,70 @@
+package rpc
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	ctypes "github.com/cometbft/cometbft/rpc/core/types"
+	"github.com/cosmos/cosmos-sdk/types/module/testutil"
+	"github.com/umee-network/umee/v6/sdkclient/query"
+	otypes "github.com/umee-network/umee/v6/x/oracle/types"
+)
+
+func GetAcceptedDenoms(codec testutil.TestEncodingConfig, grpcEndpoint string) []string {
+	logger := log.New(os.Stderr, "chain-client", log.LstdFlags)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+
+	queryClient, err := query.NewClient(logger, grpcEndpoint, 15*time.Second)
+	if err != nil {
+		log.Fatalf("Failed to initalize the query client %s and Error:  %s", grpcEndpoint, err.Error())
+	}
+
+	oQueryClient := otypes.NewQueryClient(queryClient.GrpcConn)
```

Comment:
let's move the client creation (lines 22-27) to a separate function

Reply:
OK